ts/hooks/useParamSelector.ts

```diff
@@ -183,3 +183,61 @@ export function useMessageReactsPropsById(messageId?: string) {
     return messageReactsProps;
   });
 }
+
+// TODO remove 10 seconds timer
+export function useTimerOptionsByMode(disappearingMessageMode?: string, hasOnlyOneMode?: boolean) {
+  return useSelector((state: StateType) => {
+    const options = state.timerOptions.timerOptions;
+
+    switch (disappearingMessageMode) {
+      // TODO legacy messages support will be removed in a future release
+      case 'legacy':
+        return options.filter(option => {
+          return (
+            (hasOnlyOneMode && option.value === 0) ||
+            option.value === 5 || // 5 seconds
+            option.value === 10 || // 10 seconds
+            option.value === 30 || // 30 seconds
+            option.value === 60 || // 1 minute
+            option.value === 300 || // 5 minutes
+            option.value === 1800 || // 30 minutes
+            option.value === 3600 || // 1 hour
+            option.value === 21600 || // 6 hours
+            option.value === 43200 || // 12 hours
+            option.value === 86400 || // 1 day
+            option.value === 604800 // 1 week
+          );
+        });
+      case 'deleteAfterSend':
+        return options.filter(option => {
+          return (
+            (hasOnlyOneMode && option.value === 0) ||
+            option.value === 10 || // 10 seconds (for development)
+            option.value === 30 || // 30 seconds (for development)
+            option.value === 60 || // 1 minute (for testing)
```

Comment:
might be worth to make those dependent on process.env through a feature flag or something

Reply:
Should we use `NODE_APP_INSTANCE`?

Otherwise this could be a good time to add a `yarn dev`. Something like this?

```
"dev": "cross-env NODE_ENV=development NODE_APP_INSTANCE=devprod$MULTI electron ."
```

Reply:
Let's keep it like this until merged with user config

Reply:
For now I have just commented them out for now and lest 1 minute for q/a